src/book/02-system/01-foundations/03-continuous-integration.mdx

```diff
@@ -49,6 +49,28 @@ To view the logs for a build, scroll to the build checks at the bottom of the PR
 
 ![Screenshot of the build checks section of the PR page, with arrow pointing to primary link](./images/ci-build-logs.png 'Link to the build details with logs')
 
+## Pacman Package Cache
+
+When Buildkite receives a webhook to build and test a PR, the Buildkite agents pulls down the code and starts building a Docker image based off the Dockerfile in that branch.
+
+In the process of building our docker image, serveral [pacman](https://wiki.archlinux.org/index.php/pacman) packages are downloaded from public mirrors (if that specific [layer](https://docs.docker.com/storage/storagedriver/#images-and-layers) itself is not cached). Because these packages are downloaded numerous times, across multiple PR's, caching them locally can speed up build times and save network bandwidth.
+
+It is also possible that the [signature](https://wiki.archlinux.org/index.php/Pacman/Package_signing) on a package that is located on a public mirror changes. This change of signature can cause issues for us because we download version tagged packages and still expect an older signature. Caching these packages locally may alleviate these issues.
+
+To cache these packages locally, a [pacman](https://wiki.archlinux.org/index.php/pacman) package cache is setup on each buildkite agent. This cache uses the [nroi/flexo](https://hub.docker.com/r/nroi/flexo) docker image, and exposes itself on
+
+```
+http://localhost:7878/$repo/os/$arch
```

Comment:
Perhaps add a line below saying what `$repo` and `$arch` will be expanded to.

Reply:
Honestly not too sure what these expand to, they don't seem to be set as environment variables inside arch or our container, but they definitely do need to be in the URL.

Reply:
Arch linux/pacman expands them internally. `$arch` is typically (and in our case will be) `x86_64`. `$repo` is probably one of these (and possibly others)
 - community
 - community-staging
 - community-testing
 - core
 - extra
 - multilib
 - multilib-staging
 - multilib-testing
 - staging
 - testing

Reply:
Since these are expanded internally by pacman and it isnt something we are concered with (plus couldnt find any documentation on them :stuck_out_tongue:), I dont think its necessary to add anything.